### ISSUE TITLE
Samsung Gear 360

### ISSUE THREAD

**Issue by paulinhop** [JeremyHeleine/Photo-Sphere-Viewer#35]
Hello,

Im trying out photo-sphere-viewer solution to embed the photos produced by samsung gear 360 in my webpage.
The problem is the photomerging isnt right, for some reason the 2 pictures have some black spots in it.  Please check the attachments, is it a bug? maybe im doing something wrong?

![foto1](https://cloud.githubusercontent.com/assets/17089365/16548783/17b2b6de-416d-11e6-99a3-ff668432ebcf.jpg)
![foto2](https://cloud.githubusercontent.com/assets/17089365/16548784/17b4ec92-416d-11e6-8766-1d565e0bd029.jpg)

**Comment by paulinhop** [comment]
Btw, im using a basic code, this is my code:  I also attached now the image im using.



<link rel="stylesheet" href="phpClasses/PhotoSphere/dist/photo-sphere-viewer.min.css">

<script src="phpClasses/three.js/build/three.min.js"></script>
<script src="phpClasses/D.js/lib/D.min.js"></script>
<script src="phpClasses/uevent/uevent.min.js"></script>
<script src="phpClasses/doT/doT.min.js"></script>
<script src="phpClasses/PhotoSphere/dist/photo-sphere-viewer.min.js"></script>


</head>

<body>
Teste de Imagem 360º


    <div id="foto_panoramica">
    	
    
    
    
    </div>
    
    
<script>
  var viewer = PhotoSphereViewer({ 
  panorama: 'temp/360_0004.JPG',
  container: 'foto_panoramica',
	caption: '<b>&copy; SAS</b>',
    loading_img: 'http://photo-sphere-viewer.js.org/assets/photosphere-logo.gif',
    navbar: 'autorotate zoom download caption fullscreen',
    default_fov: 70,
    mousewheel: false,
    size: {
      height: 500,
	  width: 1000
    }  });
  
</script>


![360_0004](https://cloud.githubusercontent.com/assets/17089365/16548873/19190a5e-416e-11e6-9ef2-7da1ddee8d50.JPG)

**Comment by JeremyHeleine** [comment]
Hi,

I'm not sure it's a bug. Actually we "see" that black areas in the original image. I never tried the Samsung Gear 360, is it its default output?

**Comment by paulinhop** [comment]
Hello Jeremy,

Thank you for your answer, i just found out 2 min. ago that it isnt a bug, its actually what you are going to with your question.  Its the output format.  The biggest problem is, when i take several pictures, in the mircro sd card it saves with the format i showed in my message.  If i select the picture through the gear 360 software in my phone i can change the output to panorama and it will work fine.  The problem with this is i have to do this picture by picture instead of making it like a bulk operation directly from the micro sd card.

**Comment by JeremyHeleine** [comment]
As I said, I never tried the Samsung Gear 360 so I won't be able to help you for that part. Maybe you can have a chance to automate the conversions if Samsung provides a command line tool?

**Comment by paulinhop** [comment]
Yes that would be great, but i dont think they have a command line tool.
Im trying to find a software to convert the image they save into the sd card, witch is like the one i showed, and transform it in a standard panorama image that i can use anywhere, including your script.

**Comment by claudiababescu** [comment]
If you're on windows try [Samsung Gear 360 Action Director ](https://resources.samsungdevelopers.com/Gear_360/02_Download_Gear_360_Action_Director). It will stitch the photos for you from source.

**Comment by paulinhop** [comment]
Hello @claudiababescu ,

May i ask you how?  I have the gear 360 action director, but i isnt it only for videos?
Can you point me where is the "conversion" of the photos so they stitch in the right format?

**Comment by claudiababescu** [comment]
Hi, Action Director is for both photos and videos. I wrote a tutorial on how to stitch 360 photos with Action Director here : https://www.startintech.co.uk/how-to-stitch-360-photos/ . Hope it works for you.

**Comment by ftoledo** [comment]
Hello all, 
I get a Gear 360 few days ago..i see that the output format are jpg with the two lens images together.
The phone software (Samsung Gear Admin for android or Action Director for pc) process the photo previous to "download" from the camera to the phone (or pc). Unfortunately this app run only on certain devices (my phone S4 are not supported and i had to use a modified version of this app that do not work very well for testing)

I'm are gnu/linux user and seach for free software to process (stitch?) the photo that download and use directly from sdcard. 
I'm found hugin (http://hugin.sourceforge.net/), i will investigate it.
Also, found this scripts https://github.com/ultramango/gear360pano/ whitout good result yet (i'm newbie on "360" world")
If you viewer could do this process, it will be fantastic.

Example images: 
Original:
http://bbs.docksud.com.ar/tmp/360/360_0019.JPG

Stitched from androiod gear app:
http://bbs.docksud.com.ar/tmp/360/1483418695919.jpg

**Comment by nunesbns** [comment]
Import that image on Gear 360 ActionDirector, right click on image thumb and open file path. Use the image without the black borders. Done!
